crud/stats/metrics_registry.lua

```diff
@@ -0,0 +1,232 @@
+local is_package, metrics = pcall(require, 'metrics')
+
+local label = require('crud.stats.label')
+local dev_checks = require('crud.common.dev_checks')
+local registry_common = require('crud.stats.registry_common')
+
+local registry = {}
+local _registry = {}
```

Comment:
Let's call it internal_registry or something else. It could confuse because "_registry" and "registry" is too similar and there are no comments that describes difference.

Reply:
Reworked